docs/HowToGuides/GettingStarted.md

```diff
@@ -190,6 +190,9 @@ toolchain as a one-off, there are a couple of differences:
 > LLDB currently requires at least `swig-1.3.40` but will successfully build
 > with version 2 shipped with Ubuntu.
 
+> **Note**  
+> When building on Linux, make sure to have at least 8 GB when using flag `--release-debuginfo` and 32 GB when using flag `--debug-swift` of RAM and 150 GB of disk space available.
```

Comment:
While it may be a bit redundant, it reads better if you write "GB of RAM" twice, rather than tuck that away at the end.

Reply:
Thank you, improved! 🔥